response/settings/base.py

```diff
@@ -191,19 +191,39 @@ def get_user_id(user_name, token):
 
 
 def get_channel_id(channel_name, token):
+    next_cursor = None
     slack_client = SlackClient(token)
-    response = slack_client.api_call(
-        "channels.list",
-        exclude_archived=True,
-        exclude_members=True,
-    )
-    if not response.get("ok", False):
-        raise ImproperlyConfigured(f"Failed to get channel id for \"{channel_name}\" : {response['error']}")
-
-    for channel in response['channels']:
-        if channel['name'] == channel_name:
-            return channel['id']
 
+    print(f"get_channel_id - Searching for {channel_name} id...")
```

Comment:
For the debug logging statements, would we want to use the logger, so it can be logged as debug log level, and disabled unless needed for debugging purposes?

Here's an [example](https://github.com/monzo/response/blob/fe58f146f4a8c16466d7199ca3b687889b762662/slack/models/comms_channel.py#L23) from the comms channel file, if it can be applied here?

Reply:
Right. `logger` is already included. But when I change the statements to logger statements, they are no longer sent to STDOUT, even when I change the level to INFO in my .env:

```
DJANGO_LOG_LEVEL=INFO
```

I'll make the PR as you ask, but keep in mind it does make all useful debugging output disappear into a void.